### PR TITLE
Update docs/admin/authentication.md

### PR DESCRIPTION
**Pull request by islinwb**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5408)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ with a value of `Basic BASE64ENCODED(USER:PASSWORD)`.
 
 ### Service Account Tokens
 
-Service accounts are an automatically enabled authenticator that uses signed
+Service account is an automatically enabled authenticator that uses signed
```
Comment (zacharysarah) [inline docs/admin/authentication.md]: Change to:
```
A service account is...
```

**Comment by islinwb** [comment]
@zacharysarah  Changes has been made. Thanks. Actually I've checked the box before but you still couldn't make the change. I have no idea what was wrong. Do I need to do more than just checking the box?

**Comment by islinwb** [comment]
Update: 
@zacharysarah I guess another box (after creating the PR) needs to be ckecked other than the box appeard when creating PR. The two boxes are of the same content. It's on the right side of the PR page.

**Comment by zacharysarah** [comment]
@islinwb 👋 

> The two boxes are of the same content. It's on the right side of the PR page.

Thank you for solving that question! The process to allow edits from maintainers is more confusing that it should be; I will see what we can do about making it easier and more straightforward. 

/cc @kubernetes/sig-docs-maintainers for visibility

**Pull request merged**